docs/admin-sdk/docs/guide/2_api-reference/toast.md

```diff
@@ -0,0 +1,34 @@
+# Toast
+
+### Dispatch a toast
```

Comment:
I would also add a information at which Shopware version this feature is available.